src/omnicore/rpctxobject.cpp

```diff
@@ -241,6 +244,11 @@ void populateRPCTypeSendToOwners(CMPTransaction& omniObj, Object& txobj, bool ex
     if (extendedDetails) populateRPCExtendedTypeSendToOwners(omniObj.getHash(), extendedDetailsFilter, txobj);
 }
 
+void populateRPCTypeSendAll(CMPTransaction& omniObj, Object& txobj)
+{
+    // TODO: list all recipients?
```

Comment:
We'll need to store that data to allow later retrieval, I'll take & look & suggest something :)